src/explore-education-statistics-frontend/src/modules/find-statistics/components/PublicationList.tsx

```diff
@@ -4,19 +4,20 @@ import React from 'react';
 
 interface Props {
   publications: PublicationSummary[];
+  testId?: string;
 }
 
-function PublicationList({ publications }: Props) {
+function PublicationList({ publications, testId }: Props) {
   return publications.length > 0 ? (
-    <ul className="govuk-!-margin-top-0">
+    <ul className="govuk-!-margin-top-0" data-testid={testId}>
       {publications.map(({ id, legacyPublicationUrl, slug, title }) => (
         <li key={id} className="govuk-!-margin-bottom-0">
-          <h3
-            id={`publication-heading-${id}}`}
-            className="govuk-heading-s govuk-!-margin-bottom-0"
+          <h4
+            id={`publication-heading-${id}`}
+            className="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-bottom-0"
```

Comment:
Is it a good idea to make this heading non-bold? Kinda feels to me like it blending into the rest of the text. 

Were you worried it might be indistinguishable from the publication type heading? In which case, it might better to make that a bit bigger like so:

![image](https://user-images.githubusercontent.com/9917868/152384470-4b12f9b3-695a-4373-bc2b-af27b8ef540c.png)

Reply:
Yes, I went through a few iterations, one where they were both the same size and bold and they couldn't be distinguished like you say.

The suggestion to make the Publication title not bold came from @cjrace.

I've changed it how you suggested and will re-issue the example screenshot to @cjrace and @lauraselby, thanks.

Reply:
Update: Spoke to @lauraselby who in turn has checked with @cjrace and they prefer the original so I have changed it back.